erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -468,6 +468,7 @@ def create_work_order(self, item):
 			wo.fg_warehouse = warehouse.get('fg_warehouse')
 		try:
 			wo.flags.ignore_mandatory = True
+			wo.flags.ignore_validate = True
```

Comment:
Why ignore validate? This will also skip other methods:

https://github.com/frappe/erpnext/blob/f1b77360ee3b5cc418f074aefa83a1871964da99/erpnext/manufacturing/doctype/work_order/work_order.py#L41-L55

Reply:
Fixed, added to test some cases and then forgot to remove it.